README.md

```diff
@@ -1,40 +1,86 @@
 # Lab 06
 
+Now that we have some basics down, let's create a new feature. 
+
+1. We are going to add an alert bar to notify my readers of interesting content. First, modify the layout to have a full-width section at the top called `promo` for the alert bar.
+
+2. Create a new feature in the `global` directory called `alert-bar`. 
+
+3. [Check out the ALC docs on custom fields.](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/api/feature-pack/components/custom-fields.md) Give the alert-bar a dropdown custom field (`PropTypes.oneOf`) with three options: breaking news, warning, and info. 
+
+```
+alertType: PropTypes.oneOf([
+      'danger', 'warning', 'info',
+    ]).tag({
+      defaultValue: 'danger',
+      labels: {
+        danger: 'breaking',
+        warning: 'warning',
+        info: 'info',
+      },
+      name: 'Alert Type',
+    })
+```
+
+4. Create a **required** custom field that allows the user to add text. Give this custom field a description that will be visible to the user in the PageBuilder admin.  
+
+```
+text: PropTypes.string.tag({
+      name: 'Main Alert Text'
+    }).isRequired
+```
+
+5. Create two custom fields to allow the user to add an external link. One field should be for the link text; the other for the url. This second one should be `PropTypes.url`, which validates the url in PageBuilder. Put these two in one **group** in the admin. 
+
+```
+linkText: PropTypes.string.tag({
+      group: 'Call to Action',
+      name: 'Text',
+      description: 'Text to prompt the users to click the link',
+    }),
+    link: PropTypes.url.tag({
+      group: 'Call to Action',
+      name: 'Url',
+    })
+```
```

Comment:
on this code block and the 2 before it, can we fix the indenting? so the first line has the same indent as it's closing brackets?